src/Sulu/Bundle/SecurityBundle/Security/Exception/InvalidTokenException.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Security\Exception;
+
+/**
+ * This Exception is thrown if a request with a not existing token tries to reset a password
+ * @package Sulu\Bundle\SecurityBundle\Security\Exception
+ */
+class InvalidTokenException extends SecurityException
+{
+
+    private $token;
+
+    public function __construct($token)
+    {
+        $this->token = $token;
+        parent::__construct('This token does not exist!', 1005);
```

Comment:
Please add the token to the message like `sprintf('The token "%s" does not exist!', $token)`